src/scenes/home/codeSchools/approvedSchools/approvedSchools.js

```diff
@@ -74,7 +74,19 @@ class ApprovedSchools extends Component {
 }
 
 ApprovedSchools.propTypes = {
-  schools: PropTypes.array.isRequired // eslint-disable-line
+  schools: PropTypes.arrayOf(PropTypes.shape({
+    created_at: PropTypes.string,
+    full_time: PropTypes.bool,
+    hardware_included: PropTypes.bool,
+    has_onlin: PropTypes.bool,
```

Comment:
typo here - should be: `has_online`

Looks like the typo is in each of the sections, probably a copy-pasta error.